content/libraries/ruby.mdx

```diff
@@ -0,0 +1,56 @@
+---
+title: Ruby
+description: The Official Top.gg ruby library
```

Comment:
The only official Top.gg libraries are for Python and Javascript. I know the rest of the docs site is misleading about that and I will be changing that soon. Please mark this as community-maintained and not official as we don't have time to go through every single change in every SDK

Reply:
alright sure!